hydroflow/src/scheduled/graph.rs

```diff
@@ -409,8 +412,79 @@ impl Hydroflow {
             _phantom: PhantomData,
         }
     }
+
+    pub fn add_dependencies(&mut self, subg_id: SubgraphId, deps: DirectedEdgeSet) {
+        let base = self.subgraphs[subg_id].dependencies.current_id;
+        for (key, val) in deps.node_names {
+            self.subgraphs[subg_id]
+                .dependencies
+                .node_names
+                .insert(key + base, val);
+        }
+        for (from, to) in deps.edges {
+            self.subgraphs[subg_id]
+                .dependencies
+                .edges
+                .insert((from + base, to + base));
+        }
+        self.subgraphs[subg_id].dependencies.current_id += deps.current_id;
+    }
+
+    fn mermaid_mangle(&self, name: String, subg: usize, node: u16) -> String {
+        let res = if name.starts_with("Handoff") {
+            let handoff_id: usize = name.split('_').collect::<Vec<&str>>()[1].parse().unwrap();
+            let handoff = &self.handoffs[handoff_id];
+            // let name = self.remove_whitespace(handoff.name.clone());
+            format!("Handoff:_{}[\\{}/]", handoff_id, handoff.name)
+            // name
+        } else if name.as_str() == "PullToPush" {
+            format!("{}.{}[/{}\\]", subg, node, name)
+        } else {
+            // format!("{}.{}({})", subg, node, name)
+            format!("{}.{}[{}]", subg, node, name)
+        };
+        res
```

Comment:
Remove one-off `res` variable, just have the branches without a semicolon at the end.

Also remove stray comments